src/test/java/org/jpeek/web/DyNumTest.java

```diff
@@ -41,56 +41,51 @@
     @Test
     public void testConstructDyNumFromString() {
         final Double number = 2.0d;
-        final DyNum dynum = new DyNum(
-            String.valueOf(number)
-        );
         new Assertion<>(
             "Double values should be equal",
-            dynum::doubleValue,
+            new DyNum(
+                String.valueOf(number)
+            )::doubleValue,
             new IsEqual<>(number)
         ).affirm();
     }
 
     @Test
     public void testConstructDyNumFromLong() {
         final Long number = 2L;
```

Comment:
can be primitive, just `long`

Reply:
It's needed to cover these lines in LoC limit. This issue is about this tech debt of the project and strange behaviour when commit without code changes can drop the LoC limit below configured 0.67.